### DISCUSSION TITLE
Zoom on Mermaid diagrams

### DISCUSSION THREAD

**Discussion by benjaoming** [GitbookIO/community#340] [Feature Requests]
Have you tried using Mermaid diagrams that are a little more complex? Like a sequence diagram? You can't see things when they're only allowed to have ~900px of screen width.

You need to be able to zoom.

Otherwise these diagrams are close to useless.

Try to upload a .mermaid file on GitHub and see how it displays it. It has navigation and zoom controls:

![image](https://github.com/GitbookIO/community/assets/374612/4e34bb67-0fdf-4d0d-8b30-7d3ee7843448)

**Comment by AleksandraSwiderska** [comment]
The mermaid integration is really cool and convenient. But, with more complex diagrams, which we use frequently, it is hard to make them readable on a page as the content becomes quite small. It would be great if the diagrams could be enlarged with a click, just as the images are. It is a really needed feature.

**Comment by AsaEkengren** [comment]
Agreed. We are having the same issue, which makes use of Mermaid much more limited.

**Comment by taranvohra** [comment]
Hello all. We released zooming and panning of mermaid diagrams both in the app and in published content yesterday! 

I'll be closing this discussion.

**Reply by AleksandraSwiderska** [reply]
That is great to hear. But I can't actually see that on our end. Do we need to enable that somehow?

**Reply by taranvohra** [reply]
It's enabled automatically. Can you go to the app and make a small change on the page where the mermaid block is. You might be getting a cached page
